passes/techmap/abc.cc

```diff
@@ -849,6 +861,7 @@ void abc_module(RTLIL::Design *design, RTLIL::Module *current_module, std::strin
 		for (size_t pos = abc_script.find("dretime;"); pos != std::string::npos; pos = abc_script.find("dretime;", pos+1))
 			abc_script = abc_script.substr(0, pos) + "dretime; retime -o {D};" + abc_script.substr(pos+8);
 
+	// replace placeholders in abc.script and user.script
```

Comment:
Please factor out the substitution operation so that it's not open-coded many times